### ISSUE TITLE
release 0.1.2

### ISSUE THREAD

**Issue by plandolt** [disconn3ct/cmk-rabbitmq#9]
with the fixes you should release a new version. thank you!

**Comment by eetievent** [comment]
Indeed, the fix about changing first to dict in web/plugins/wato/rabbitmq_queues_check_parameters.py is not in the mkp here:
https://checkmk.com/check_mk-exchange-file.php?&file=rabbitmq-0.1.1.mkp
